config/cache.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+use Illuminate\Support\Str;
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Default Cache Store
+    |--------------------------------------------------------------------------
+    |
+    | This option controls the default cache connection that gets used while
+    | using this caching library. This connection is used when another is
+    | not explicitly specified when executing a given caching function.
+    |
+    */
+
+    'default' => env('CACHE_DRIVER', 'momento'),
```

Comment:
I think we want to add another comment here to call attention to the importance of this line vs. a default laravel app.  Or modify the comment above.